GVRf/Extensions/debugwebserver/src/main/res/raw/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.terminal/0.10.12/js/jquery.terminal.min.js"></script>
+<link href="https://cdnjs.cloudflare.com/ajax/libs/jquery.terminal/0.10.12/css/jquery.terminal.min.css" rel="stylesheet"/>    
+<script type="text/javascript">
+
+var removePrompt = function(input, prompt) {
+    while (input.endsWith(prompt)) {
+        input = input.substring(0, input.length - prompt.length);
+    }
+    return input;
+}
+
+jQuery(document).ready(function($) {
+
+    var GVRF_PROMPT = "gvrf> ";
```

Comment:
Is it possible to avoid handling prompts locally? The server will send the prompts.